recipes/fair-research-login/meta.yaml

```diff
@@ -1,46 +0,0 @@
-{% set name = "fair-research-login" %}
```

Comment:
It’s okay here as it’s already been converted but in general you shouldn’t delete other recipes. The CI will handle them.